packages/nx/src/command-line/release/version.ts

```diff
@@ -73,6 +73,12 @@ export interface ReleaseVersionGeneratorSchema {
   installArgs?: string;
   installIgnoreScripts?: boolean;
   conventionalCommitsConfig?: NxReleaseConfig['conventionalCommits'];
+  updateDependents?: {
+    // "auto" means "only when the dependents are already included in the current batch", and is the default
+    when?: 'auto' | 'always';
```

Comment:
```suggestion
    when?: 'never' | 'auto';
```

`never` is the current behavior. No dependents are updated. 
`auto` is the new behavior and will remain ambiguous with what we think is best.

We may add more in the future.